test/src/test/java/jenkins/model/JenkinsBuildsAndWorkspacesDirectoriesTest.java

```diff
@@ -258,6 +286,13 @@ private boolean logWasFound(String searched) {
                 .anyMatch(record -> record.getMessage().contains(searched));
     }
 
+	private boolean logWasFoundAtLevel(String searched, Level level) {
```

Comment:
NIT: `logWasFound` body above should probably be rewritten to `logWasFoundAtLevel(searched, Level.ALL)`

Reply:
my bad, ^ is wrong, because it's an exact level match. So it's not fully as straightforward as I was thinking.